client/state/domains/management/reducer.js

```diff
@@ -32,20 +32,18 @@ import {
  * @param  {Object} action Action payload
  * @return {Object}        Updated state
  */
-export const _contactDetailsCache = createReducer( {}, {
+export const isRequestingContactDetails = createReducer( false, {
```

Comment:
"ContactDetailsCache" is clumsy, but "ContactDetails" gets quickly too ambiguous in usage. I consider the clumsy extra verboseness worth it for the clarity of "Cache" vs live WHOIS data so I'm going to tack "Cache" back on here @yoavf unless you strenuously object.

Splitting the `isRequesting` up seems fine.